spec/models/work_spec.rb

```diff
@@ -6,8 +6,7 @@
   let(:collection) { Collection.research_data }
   let(:user_other) { FactoryBot.create :user }
   let(:superadmin_user) { User.from_cas(OmniAuth::AuthHash.new(provider: "cas", uid: "fake1", extra: { mail: "fake@princeton.edu" })) }
-  let(:doi) { "https://doi.org/10.34770/0q6b-cj27" }
-  let(:work) { FactoryBot.create(:draft_work, doi: doi) }
+  let(:work) { FactoryBot.create(:draft_work) }
```

Comment:
You can add a transient attribute like https://github.com/pulibrary/pdc_describe/blob/main/spec/factories/resource.rb#L5-L7 To allow for setting the doi on the factory.  Not needed since the tests still pass...

Reply:
Good to know, thanks!